cmd/dao_tests/local.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
did you mean to commit this file? :thinking:

Reply:
Oh no, luckily it only contains my hostname and database name :)